samcli/cli/command.py

```diff
@@ -62,6 +62,15 @@ def __init__(self, *args, cmd_packages=None, **kwargs):
 
         if not cmd_packages:
             cmd_packages = _SAM_CLI_COMMAND_PACKAGES
+        elif isinstance(cmd_packages, list):
```

Comment:
This is in prod code. If we somehow missed importing a new command and the unit tests didn't catch it, users will see this message and have no idea what it's talking about.

Reply:
This else will only execute is `cmd_packages` is set. If that happens (which seems unlikely), then none of the commands would (potentially) work. `cmd_packages` was added to all injection, however, I think this is mainly for test reasons at this time. I don't see a strong need for us to be doing this else, unless I am missing something (which is very much possible).

Reply:
There is no usage of `cmd_packages` now except for some test cases. Here I added this else case to avoid using it without updating package in the future. I will remove this.